boards.txt

```diff
@@ -6233,7 +6235,170 @@ twatch_ultra.menu.Revision.Radio_SX1280=Radio-SX1280
 twatch_ultra.menu.Revision.Radio_SX1280.build.board=LILYGO_LORA_SX1280
 twatch_ultra.menu.Revision.Radio_SX1262=Radio-SX1262
 twatch_ultra.menu.Revision.Radio_SX1262.build.board=LILYGO_LORA_SX1262
+twatch_ultra.menu.Revision.Radio_CC1101=Radio-CC1101
+twatch_ultra.menu.Revision.Radio_CC1101.build.board=LILYGO_LORA_CC1101
+
+
+##############################################################
+
+tlora_pager.name=LilyGo-T-LoRa-Pager
+
+tlora_pager.bootloader.tool=esptool_py
+tlora_pager.bootloader.tool.default=esptool_py
+
+tlora_pager.upload.tool=esptool_py
+tlora_pager.upload.tool.default=esptool_py
+tlora_pager.upload.tool.network=esp_ota
+
+tlora_pager.upload.maximum_size=1310720
+tlora_pager.upload.maximum_data_size=327680
+tlora_pager.upload.flags=
+tlora_pager.upload.extra_flags=
+tlora_pager.upload.use_1200bps_touch=false
+tlora_pager.upload.wait_for_upload_port=false
+
+tlora_pager.serial.disableDTR=false
+tlora_pager.serial.disableRTS=false
+
+tlora_pager.build.tarch=xtensa
+tlora_pager.build.bootloader_addr=0x0
+tlora_pager.build.target=esp32s3
+tlora_pager.build.mcu=esp32s3
+tlora_pager.build.core=esp32
+tlora_pager.build.variant=lilygo_tlora_pager
+tlora_pager.build.board=T_LORA_PAGER
+
+tlora_pager.build.usb_mode=1
+tlora_pager.build.cdc_on_boot=1
+tlora_pager.build.msc_on_boot=0
+tlora_pager.build.dfu_on_boot=0
+tlora_pager.build.f_cpu=240000000L
+tlora_pager.build.flash_size=16MB
+tlora_pager.build.flash_freq=80m
+tlora_pager.build.flash_mode=dio
+tlora_pager.build.boot=qio
+tlora_pager.build.boot_freq=80m
+tlora_pager.build.partitions=app3M_fat9M_16MB
```

Comment:
This partition is overwritten, as in the partition scheme menu the 1st option is always checked, and its fat flash. Take a look which one should be the default one.